ui/src/components/building-block/OurValues.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Box, Heading, Text, Stack, Image } from '@chakra-ui/react';
+
+const values = [
+  {
+    title: 'Direction',
+    description:
+      "Children to come to a saving knowledge of Jesus Christ and be rooted in God's Word and the gospel.",
+    icon: '/images/building-blocks/DirectionIcon.png',
+    bg: '#DBF1F1',
+    shadow: '0.625rem 0.625rem 0 #77C2C6',
+  },
+  {
+    title: 'Delight',
+    description:
+      'Children to experience the joy of being in an intimate relationship with Jesus and the church community.',
+    icon: '/images/building-blocks/DelightIcon.png',
+    bg: '#F5F0E0',
+    shadow: '0.5rem 0.5rem 0 #FFE29A',
+  },
+  {
+    title: 'Destiny',
+    description:
+      "Children to grow as a disciple of Jesus, to daily live out God's call in their lives, and raise up future disciples of Christ.",
+    icon: '/images/building-blocks/DestinyIcon.png',
+    bg: '#F1E6E6',
+    shadow: '0.5rem 0.5rem 0 #F7B2B7',
+  },
+];
+
+const OurValues = () => (
+  <Box maxW="90rem" p={{ base: '1rem', md: '2rem' }} mb={20}>
+    <Heading
+      as="h1"
+      size="xl"
+      mb="0.5rem"
+      fontSize={{ base: '2.25rem', lg: '2.625rem' }}
+      fontFamily="DMSerifDisplay_Italic"
+      fontWeight="400"
+    >
+      Our Values
+    </Heading>
+    <Text
+      fontSize={{ base: '0.875rem', lg: '1.25rem' }}
+      fontFamily="Manrope"
+      mb="2.5rem"
+      fontWeight="400"
+    >
+      We hope to accomplish our mission with the 3-Ds. Our prayer is for...
+    </Text>
+    <Stack
+      direction={{ base: 'column', xl: 'row' }}
```

Comment:
i think can change to lg: 'row' - iPad Pro width (lg) should be wide enough to display the 3 blocks in a row

Reply:
![Screenshot 2025-06-01 at 12 51 10 AM](https://github.com/user-attachments/assets/3276a8a3-c28d-43bf-b6d6-31ad8e89819c)

vs.

![Screenshot 2025-06-01 at 12 51 50 AM](https://github.com/user-attachments/assets/d0b1a9b6-2423-42d1-8b7b-f08b1deb5f39)